dap-netcore.el

```diff
@@ -85,6 +85,24 @@ The slash is expected at the end."
     ("attach"
      (dap--put-if-absent conf :processId (string-to-number (read-string "Enter PID: " "2345"))))))
 
+(if (and (fboundp 'libxml-available-p)
```

Comment:
`when`  instead of `if` for if clause without else part

Reply:
fixed, thanks